pimd/pim_cmd_common.c

```diff
@@ -2840,3 +2840,44 @@ int gm_process_no_last_member_query_interval_cmd(struct vty *vty)
 	return nb_cli_apply_changes(vty, FRR_GMP_INTERFACE_XPATH,
 				    FRR_PIM_AF_XPATH_VAL);
 }
+
+
+int pim_process_ssmpingd_cmd(struct vty *vty, const char *src_str)
```

Comment:
both the functions pim_process_ssmpingd_cmd and pim_process_no_ssmpingd_cmd is exactly the same. NB_OP_CREATE and NB_OP_DESTROY is the only difference, can we pass this as a parameter and write only one function?

Reply:
Ack